tests/components/smartthings/test_light.py

```diff
@@ -451,3 +451,38 @@ async def test_availability_at_start(
     """Test unavailable at boot."""
     await setup_integration(hass, mock_config_entry)
     assert hass.states.get("light.standing_light").state == STATE_UNAVAILABLE
+
+
+@pytest.mark.parametrize("device_fixture", ["da_ks_hood_01001"])
+async def test_hood_multi_component_lights(
+    hass: HomeAssistant,
+    devices: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test that range hood device creates light entities for multiple components."""
```

Comment:
To test whether sensors are being created, we have snapshot tests. This test should check the behaviour of the `SmartThingsLight` class. In this case, it should be named "Test to check if non-main components are supported" or something similar.